src/vault/client/http.clj

```diff
@@ -91,7 +91,9 @@
              (str "API path must be a non-empty string, got: "
                   (pr-str path)))))
   ; Check client authentication.
-  (when-not (some-> client :auth deref :client-token)
+  (when-not (or
+              (some-> req :headers keys set (contains? "X-Vault-Token"))
```

Comment:
🏌️ 
```clojure
(contains? (:headers req) "X-Vault-Token")
```

Reply:
`(contains? (keys (:headers req)) "X-Vault-Token")`, done.

Reply:
thanks for helping me avoid death by syntactic sugar!
https://www.youtube.com/watch?v=ILIvPzyK_8I